### PR TITLE
[emby] New chart

### PR DESCRIPTION
**Pull request by nicholaswilde**
Signed-off-by: Nicholas Wilde <ncwilde43@gmail.com>

**Description of the change** Emby

**Benefits**

New chart

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by onedr0p** [comment]
Incubator is meant for long lived testing of a chart with pretty non-standard config. This looks as if it's fine being release as stable.

File: charts/stable/emby/values.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+service:
+  port:
+    port: 8096
+  additionalPorts:
+    - port: 8920
+      name: https
+      protocol: TCP
+
+ingress:
+  enabled: false
```
Comment (onedr0p) [inline charts/stable/emby/values.yaml:30]: You'll probably want to include some persistence below here for config and media.

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/stable/emby/values.yaml (hunk 1 above)
Comment (nicholaswilde) [reply charts/stable/emby/values.yaml:30]: I'm going to have to look into this a bit more. [Truecharts'](https://github.com/truecharts/apps/blob/56211222bff4333fdf827d34a88c62965777277c/.containers/apps/emby/Dockerfile) didn't have any `VOLUMES` , [LSIO](https://github.com/linuxserver/docker-emby/blob/master/README.md#usage) has `/data` but I don't see how Emby is mapped to it, and the official Emby Docker file is only calling out `/mnt/` but are [not referencing](https://github.com/MediaBrowser/Emby.Build/tree/master/docker-containers) them.

```
 volumes:
      - /path/to/programdata:/config # Configuration directory
      - /path/to/tvshows:/mnt/share1 # Media directory
      - /path/to/movies:/mnt/share2 # Media directory
```

Comment (nicholaswilde) [reply charts/stable/emby/values.yaml:30]: It looks like /media already exists in the image and the storage locations are set using the emby GUI. I've also added the `/config` volume. Volume mounts seem to work without declaring `VOLUME` in the Dockerfile and so I don't think there's a need to update the image.

**Review by onedr0p — APPROVED** [review]

File: charts/stable/emby/values.yaml (hunk 1 above)
Comment (onedr0p) [reply charts/stable/emby/values.yaml:30]: Looks good man, thanks!

**Pull request merged**